docs/src/modules/ROOT/pages/introduction/introduction.adoc

```diff
@@ -1,287 +0,0 @@
-[#introduction]
-= Introduction
-:page-aliases: ../index.adoc
-:doctype: book
-:sectnums:
-:icons: font
-
-[#whatIsTimefold]
-== What is Timefold Solver?
-
-Every organization faces planning problems: providing products or services with a limited set of _constrained_ resources (employees, assets, time and money).
-Timefold Solver's <<planningAI,Planning AI>> optimizes these problems to do more business with less resources using _Constraint Satisfaction Programming_, which is part of the _<<operationsResearch,Operations Research>>_ discipline.
-
-https://timefold.ai[Timefold Solver] is a lightweight, embeddable constraint satisfaction engine which optimizes planning problems.
-It solves use cases such as:
```

Comment:
The PR description mentions that this will be re-introduced as an image.

In my opinion, information should be presented exclusively as images - text is important. For fundability and accessibility, among other reasons.

Reply:
The use cases will be re-introduced in a separate chapter. For now, I felt like it added too much details to the introduction. I would put a link to the separate chapter here but it isn't available yet (since this doc redo will take a few passes). 

I agree that information shouldn't be presented exclusively as images. You are probably right that at the very least we should include alt-text for accessibility reasons.

Reply:
I have included some text as a caption to the image. In a future version, this caption would include a link to the other use cases. 

Preview:
![image](https://github.com/user-attachments/assets/28e06f9a-c666-474d-8bd9-ca997be412f9)